tests/image/imageengine.js

```diff
@@ -147,6 +147,28 @@ describe( 'ImageEngine', () => {
 				expect( getModelData( document, { withoutSelection: true } ) )
 					.to.equal( '<image alt="alt text" src="foo.png"></image>' );
 			} );
+
+			it( 'should not convert alt attribute on non-img element', () => {
```

Comment:
Well, two things:
1. Allowing alt on div is required, that's true. Without this the test doesn't make much sense.
2. It will pass on previous version, because that was a feature of previous version too.